### PR TITLE
New cargo crates to buy

### PR DESCRIPTION
**Pull request by coiax**
Also some minor spelling fixes I noticed.

:cl: coiax
tweak: Internals Crate now contains breath masks and small tanks.
rscadd: Metal foam grenade box crate added for 1000 points
rscadd: Added two Engineering Mesons to Engineering Gear Crate
tweak: Engineering Gear now costs 1300
rscadd: Breach Shield Generator Crate added for 2500 points
rscadd: Grounding Rod Crate added for 1700 points
rscadd: PACMAN Generator Crate added for 2500
rscadd: Defibrillator Crate added for 2500
rscadd: Added more helmets to the Laser Tag Crate.
rscadd: Nanotrasen Customs report that some wide band suppliers have
been providing "alternative" firing pins. Nanotrasen reminds all
crewmembers that contraband is contraband.
rscadd: Contraband crates now appear visually similar to legitimate crates.
fix: The cargo shuttle's normal transit time has been restored.
/:cl:

Laser tag firing pins (only allow gun to be fired with laser tag armour
of the appropriate colour) added to contraband.
Special clown firing pin (prank your friends) added to contraband.

### CODE REVIEW

**Comment by AdamElTablawy** [comment]
consider that newcargo gets points really really really easily, you might want to up the cost of these

**Comment by coiax** [comment]
I based the prices off existing crates, where are the price points that you disagree with?

**Comment by Incoming5643** [comment]
So many changelog entries

**Comment by coiax** [comment]
I thought we were supposed to document changes.

**Comment by PKPenguin321** [comment]
Yeah but like you can document changes without looking like an attention whore

Be more concise

For example,
>rscadd: Nanotrasen Customs report that some wide band suppliers have
 been providing "alternative" firing pins. Nanotrasen reminds all
 crewmembers that contraband is contraband.

Is completely unnecessary

**Comment by Incoming5643** [comment]
rscadd: New crates have been added to cargo

That's literally all you need

**Comment by bgobandit** [comment]
come the fuck on, we should be encouraging fluff changelogs

**Comment by xxalpha** [comment]
No you shouldn't, but the listing of what which crates were added is welcome.

Also, who set the cargo shuttle to 1 second travel?

**Comment by bgobandit** [comment]
goofy, who else

**Comment by Anonmare** [comment]
I like the alternative firing pins tbh

**Comment by Iamgoofball** [comment]
I set the cargo shuttle transfer time to super low because pretty much all maintainers and the game design lead agreed "that shit is too fucking long". It also had positive feedback from players. This isn't getting changed back.

:+1: to the PR, but grounding rods should be cheaper.

File: code/modules/cargo/packs.dm [hunk 1]
```diff
@@ -1095,6 +1140,20 @@
 					/obj/item/clothing/head/helmet/bluetaghelm)
 	crate_name = "laser tag crate"
 
+/datum/supply_pack/misc/lasertag/pins
```
Comment (octareenroon91) [inline code/modules/cargo/packs.dm]: Please merge posthaste.

File: code/modules/cargo/packs.dm [hunk 2]
```diff
@@ -668,6 +706,13 @@
 	contains = list(/obj/machinery/iv_drip)
 	crate_name = "iv drip crate"
 
+/datum/supply_pack/medical/defibs
+	name = "Defibrillator Crate"
+	cost = 1500
+	contains = list(/obj/item/weapon/defibrillator/loaded,
+					/obj/item/weapon/defibrillator/loaded,)
+	crate_name = "defibrillator crate"
+
```
Comment (Core0verload) [inline code/modules/cargo/packs.dm]: I think that's too powerful for it's cost.

File: code/modules/cargo/packs.dm [hunk 3]
```diff
@@ -1095,6 +1140,20 @@
...
+	contains = list(/obj/item/weapon/storage/box/lasertagpins,)
+	crate_name = "laser tag firing pins crate"
+
+/datum/supply_pack/misc/clownpin
+	name = "Hilarious Firing Pin Crate"
+	cost = 5000
+	contraband = TRUE
+	contains = list(/obj/item/device/firing_pin/clown,)
+	crate_name = "hilarious firing pin crate"
+
```
Comment (Core0verload) [inline code/modules/cargo/packs.dm]: Just "crate" would be better for the contraband ones.

**Comment by AdamElTablawy** [comment]
Instant cargo is the only reason anyone plays cargo because arbitrary 4 minute delays to order anything both mean that by the time you have any points the shuttle is called and you can barely get anything out in a crisis

Keep instant cargo or set it to 15 seconds it's literally the only reason anyone plays cargo

That and getting a few hundred thousand points off of selling research disks and giving security 50 traitor crates

File: code/modules/cargo/packs.dm (hunk 2 above)
Comment (PKPenguin321) [inline code/modules/cargo/packs.dm]: agreed, make it pricier

**Comment by Core0verload** [comment]
It was me who set it to 0 seconds. I did it for testing exports and the accidentally pushed it. Nobody noticed.

**Comment by coiax** [comment]
I've renamed all the  contraband and illegal crates to share appearance and name with legitimate crates, apart from the classic "contraband" crate, which maintains the normal crate name and appearance.

**Comment by Iamgoofball** [comment]
What the fuck, fix the cargo time and make it 10 seconds.

**Comment by coiax** [comment]
> It was me who set it to 0 seconds. I did it for testing exports and the accidentally pushed it. Nobody noticed.

It was an accidental commit of a debug testing time of 1 second. I'm reverting it back.

**Comment by Iamgoofball** [comment]
>>>2 minutes

ARE YOU FUCKING RETARDED HOLY SHIT DID YOU READ ANY OF THE COMMENTS STATING "LEAVE IT AS-IS"

**Comment by jughu** [comment]
Goof was right

**Comment by Mercenaryblue** [comment]
As the creator of the laser-tag helmet, I just wanted to say they were made rare to mark the leader of the team, rather than be mass-produced for everyone. Plus, it also limit the numbers of ED-TagLaser that can be built.

**Pull request merged**